shader_source.py

```diff
@@ -0,0 +1,21 @@
+import os
+import requests
+
+
+class GetShaderCode:
```

Comment:
Maybe we should rename this to ShadertoyAPI or Shadertoy.
Prefer noun classes with verb methods, so in that case even ShaderCodeGetter would be better even though thats icky too haha